openlibrary/core/bookshelves.py

```diff
@@ -116,7 +116,7 @@ def count_total_books_logged_by_user_per_shelf(cls, username, bookshelf_ids=None
         return dict([(i['bookshelf_id'], i['count']) for i in result]) if result else {}
 
     @classmethod
-    def get_users_logged_books(cls, username, bookshelf_id=None, limit=100, page=1):
+    def get_users_logged_books(cls, username, bookshelf_id=None, limit=100, page=1, date_added=None):
```

Comment:
Let's make this a bit more explicit:

```suggestion
    def get_users_logged_books(cls, username, bookshelf_id=None, limit=100, page=1, sort: Literal[None, 'created'] = None, sort_order: Literal['asc', 'desc'] = 'asc'):
```